build-ps/percona-server.spec

```diff
@@ -190,6 +191,7 @@ Requires:       grep
 Requires:       procps
 Requires:       shadow-utils
 Requires:       net-tools
+Requires:       openldap-devel
```

Comment:
requires should be handled automatically at binary level. anyway, are you sure that development library is required for runtime?